share/spice/expand_url/expand_url.js

```diff
@@ -21,12 +22,16 @@
                 sourceUrl: "http://untiny.me/",
                 sourceName: "Untiny"
             },
+            normalize: function (item) {
+                return {
+                    title: item.org_url,
+                    url: item.org_url,
+                    description: "Expanded URL for: " + decodeURIComponent(query)
```

Comment:
I think the more correct approach would then be to use the `text` template with a `title_content.handlebars` template and a standard sub-template. It should look just like the IsItUp Spice was recently made to look like.

I'd rather we use a sub-template with built-in classes we know will work (`c-base__title`), instead of modifying the description to look like a subttle.